coworking-metz/tickets-manager-web
# feat(members): add a sidedialog to update subscriptions

## The problem

Related to #4 

## How to test it manually


- Go to http://localhost:5173/
- Sign in
- Check that everything is working as intended
